Add escapeReservedKeywords in all templates where needed

djbe opened the pull request:
Fixes #3.

Only the structured/dot-syntax string templates already had this. In some places in the storyboards templates, it isn't needed to escape the identifier because "Scene" or "ViewController" is appended to it.

djbe committed:
Add escapeReservedKeywords in all templates where needed

AliSoftware reviewed (changes requested):
We should add unit tests for that.
I mean, not really add, but just change the current fixtures so they all have at least one asset named with a reserved, and adapt the expected output to ensure it's properly escaped (and then #15 and #16 would help ensure it's valid Swift)

djbe committed:
images tests

djbe committed:
strings tests

djbe committed:
colors tests

djbe committed:
ios storyboards

djbe committed:
storyboards mac

djbe committed:
Font tests.
Custom font is based on Proza Libre (https://fontlibrary.org/en/font/proza-libre), SIL Open Font license.
